### PR TITLE
[FIXED JENKINS-36277] Check that process working dir exists

### PR DESCRIPTION
**Pull request by vjuranek**
https://issues.jenkins-ci.org/browse/JENKINS-36277

### CODE REVIEW

File: core/src/main/java/hudson/Proc.java [hunk 1]
```diff
@@ -237,6 +237,9 @@ private LocalProc( String name, ProcessBuilder procBuilder, InputStream in, Outp
             this.out = out;
             this.cookie = EnvVars.createCookie();
             procBuilder.environment().putAll(cookie);
+            if (procBuilder.directory() != null && !procBuilder.directory().exists()) {
+                throw new IOException(String.format("Process working directory '%s' doesn't exists!", procBuilder.directory().getAbsolutePath()));
```
Comment (jtnord) [inline core/src/main/java/hudson/Proc.java]: NIT "doesn't exist!"

**Comment by jtnord** [comment]
:+1:

File: core/src/main/java/hudson/Proc.java (hunk 1 above)
Comment (vjuranek) [inline core/src/main/java/hudson/Proc.java]: thanks, both typos are now fixed (same typo was in the test)

**Comment by jtnord** [comment]
:+1:

**Comment by oleg-nenashev** [comment]
:+1:

**Pull request merged**